src/components/DayPickerNavigation.jsx

```diff
@@ -72,6 +82,7 @@ export default function DayPickerNavigation(props) {
     <div className={navClassNames}>
       {!isVerticalScrollable &&
         <span
+          aria-label={phrases.jumpToPrevMonth}
```

Comment:
Yes. I figured we'd do some of the small aria-labeling/add those phrases unrelated to keyboard nav here.